src/main/js/utils/Grade.js

```diff
@@ -0,0 +1,43 @@
+import queryString from 'query-string';
+import 'whatwg-fetch';
+import { gpaSearchUrl } from '../constants/resources';
+
+const getMapping = (gradeList) => {
```

Comment:
Calculate the GPAs for each instructor here

Reply:
Done.